src/main/kotlin/com/devfloor/untitled/comment/application/CommentModifyRequest.kt

```diff
@@ -0,0 +1,6 @@
+package com.devfloor.untitled.comment.application
+
+data class CommentModifyRequest(
+    val content: String,
+) {
+}
```

Comment:
별건 아닌데 여기 아래와같이 바꿔도 될것 같아요 !! 
```suggestion
data class CommentModifyRequest(val content: String)
```

Reply:
넹!